eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -43,6 +43,9 @@ parameters:
   - name: BuildParallelization
     type: string
     default: '2C'
+  - name: AdditionalLintingOptions
```

Comment:
For a different PR, I wonder if we can get rid of this somehow. It's only used in one place (keyvault).

Reply:
I'll add it into the backlog. I think there is a lot the Java CI setup could do better given how much it has grown organically as we needed to support new changes.